tests/testing_mesh_creation_functions.py

```diff
@@ -737,6 +738,99 @@ def line(t):
         # Check the output.
         compare_test_result(self, input_file.get_string(header=False))
 
+    def test_mesh_creation_functions_helix_no_rotation(self):
+        """Create a helix and compare it with the reference file."""
+
+        # Create mesh.
+        mesh = Mesh()
+
+        # Create input file.
+        input_file = InputFile()
+
+        # Add material and function.
+        mat = MaterialReissner(youngs_modulus=1e5, radius=0.5, shear_correction=1.0)
+
+        # Create helix.
+        create_beam_mesh_helix(
+            mesh,
+            Beam3rHerm2Line3,
+            mat,
+            [0.0, 0.0, 1.0],
+            [0.0, 0.0, 0.0],
+            [2.0, 0.0, 0.0],
+            np.pi / 4,
+            height_helix=80.0,
+            l_el=5.0,
+        )
```

Comment:
You get return values from `create_beam_mesh_helix` (node sets). To also test that they are correct, write the following (this will require an update of the reference files):
```python3
helix_set = create_beam_mesh_helix(
            input_file,
            Beam3rHerm2Line3,
            mat,
            [0.0, 0.0, 1.0],
            [0.0, 0.0, 0.0],
            [2.0, 0.0, 0.0],
            np.pi / 4,
            height_helix=80.0,
            l_el=5.0,
        )
input_file.add(helix_set)
```